app/src/main/java/it/niedermann/owncloud/notes/android/fragment/NotePreviewFragment.java

```diff
@@ -19,23 +21,32 @@
 import com.yydcdut.rxmarkdown.RxMDTextView;
 import com.yydcdut.rxmarkdown.RxMarkdown;
 
+import java.util.HashSet;
+import java.util.List;
 import java.util.Objects;
+import java.util.Set;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import it.niedermann.owncloud.notes.R;
+import it.niedermann.owncloud.notes.android.activity.EditNoteActivity;
+import it.niedermann.owncloud.notes.model.DBNote;
+import it.niedermann.owncloud.notes.persistence.NoteSQLiteOpenHelper;
 import it.niedermann.owncloud.notes.util.MarkDownUtil;
+import it.niedermann.owncloud.notes.util.NoteLinksUtils;
 import rx.Subscriber;
 import rx.android.schedulers.AndroidSchedulers;
 import rx.schedulers.Schedulers;
 
 public class NotePreviewFragment extends BaseNoteFragment {
-    
+
     private static final String TAG = NotePreviewFragment.class.getSimpleName();
 
     @BindView(R.id.single_note_content)
     RxMDTextView noteContent;
 
+    private NoteSQLiteOpenHelper db = NoteSQLiteOpenHelper.getInstance(getActivity());
+
```

Comment:
There's already a `NoteSQLiteOpenHelper` instance in `BaseNoteFragment`. We should better make that `protected`.

Reply:
agreed

Reply:
please review again and resolve if you are happy with the changes